hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java

```diff
@@ -19,7 +19,6 @@
 import com.hazelcast.core.Hazelcast;
 import com.hazelcast.core.HazelcastInstance;
```

Comment:
I would copy all the testing improvements to a different PR. This way you reduce the size of this PR. And in general shorter pr's are better for everyone; committer and reviewer.